### ISSUE TITLE
Change way video->video processing pipeline works

### ISSUE THREAD

**Issue by himynamesdave** [trek-view/gopro2gsv#26]
https://github.com/trek-view/gopro2gsv#video---video-processing-pipeline

**Comment by fqrious** [comment]
Is `--keep-extracted-frames` actually needed, I believe it's better to do most of the operations in memory such that there isn't a need to geotag every single frame before running the image-video pipeline on it

**Comment by himynamesdave** [comment]
@fqrious , yes, it is needed. The reason it is optional is because sometimes a user wants to keep the geotagged frames for other purposes, outside of gopro2gsv

**Comment by fqrious** [comment]
Alright, I'll do that...

The problem here is that there is significantly less number of gps data than there are frames, it's like 1:4 ratio... I already sorted it out anyways, I just wanted to confirm

**Comment by fqrious** [comment]
If `extract_fps` isn't set, should it default to 10 or video_fps?

**Comment by himynamesdave** [comment]
@fqrious if no extract_fps value set, then we should process the video as we do now. That is, do not break it into frames and pass it on as it is.

**Comment by fqrious** [comment]
done

**Comment by himynamesdave** [comment]
Closing this as done, will raise individual issues individually.
